### DISCUSSION TITLE
WHIP 5: Expert Program

### DISCUSSION THREAD

**Discussion by chiumichael** [wihi-labs/WHIP#16] [Announcements]
- Authors: [Mark Ballandies](https://twitter.com/BallandiesMC)
- Start Date: 2023-09-28
- Category: Organization
- Tracking Issue: #6
----

## Summary

The WiHi Advisor program is a mechanism to grow WiHi’s community of active supporters (see DAO Onion in WHIP-2).

It is intended to be a pool of active individuals that regularly and pro-actively extend onocoys capabilities within the organisation.

This WHIP-5 specifies the structure, goals and mechanisms of the Advisor program.

Link to WHIP: 

https://github.com/wihi-labs/WHIP/blob/main/0005-expert-program.md

**Comment by urosvk** [comment]
ack
